styles/footer.css

```
@@ -0,0 +1,19 @@
+footer {
+  background-color: #FFBA51;
+}
+h4 {
+  font-size: 24px;
+  font-weight: bold;
+}
+p {
```

```suggestion
footer p {
```

Просто селектор `p` задает стили **всем** тегам `p` на сайте. Подозреваю, что это не то, что вам нужно.